core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -630,6 +631,31 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  private def maybeUpdateBillingAccountInGoogle(projectName: RawlsBillingProjectName, billingAccount: Option[RawlsBillingAccountName])
+      : Future[Unit] = {
+    val googleProjectId = GoogleProjectId(projectName.value)
+    for {
+      googleProjectExists <- rawlsCreatedGoogleProjectExists(googleProjectId)
+    } yield (if (googleProjectExists) gcsDAO.maybeUpdateBillingAccount(googleProjectId, billingAccount).recoverWith {
+      // TODO: Move this to a method (duplicated with monitor code)
```

Comment:
So, I was thinking about moving this to something shared, but nothing actually requires it to be the same code, and I don't know if we have a clear model for code sharing. It isn't right to put it in the GoogleServicesDAO, in my opinion.